scripts/ai/FillLevelManager.lua

```diff
@@ -28,7 +28,7 @@ function FillLevelManager:init(vehicle, debugChannel)
 end
 
 function FillLevelManager:debug(...)
-    CpUtil.debugVehicle(self.debugChannel, self.vehicle, ...)
+    CpUtil.debugVehicle(self.debugChannel or CpDebug.DBG_IMPLEMENTS, self.vehicle, ...)
 end
 
 function FillLevelManager:debugSparse(...)

```

Comment:
Should use CpUtil.debugVehicle() with an object.
@schwiti6190 : we should consider changing CpUtil.debugVehicle() to add the rootVehicle to the logging if vehicle ~= rootVehicle.